src/target/stm32f4.c

```diff
@@ -423,9 +423,21 @@ static bool stm32f4_attach(target_s *const target)
 	/* And rebuild the RAM map */
 	bool use_dual_bank = !is_f7 && dual_bank;
 	if (is_f7) {
-		target_add_ram32(target, 0x00000000, 0x4000);  /* 16kiB ITCM RAM */
-		target_add_ram32(target, 0x20000000, 0x20000); /* 128kiB DTCM RAM */
-		target_add_ram32(target, 0x20020000, 0x60000); /* 384kiB RAM */
+		uint32_t dtcm_size = 64U * 1024U; /* 64kiB DTCM RAM */
+		uint32_t ahbsram_size = 16U * 1024U;
+		if (target->part_id == ID_STM32F72X) {
+			ahbsram_size = (176U + 16U) * 1024U; /* 192kiB SRAM1/2 */
+		} else if (target->part_id == ID_STM32F74X) {
```

Comment:
This is a single statement `if` statement - drop the braces please. Same for the following `else if`.